### ISSUE TITLE
Click method to install R studio

### ISSUE THREAD

**Issue by JackSparrow1109** [Genomics-CRT/Data-Science-For-Life-Science#5]
Hi there, 
In home of Anaconda navigator , I found this:
![Rstudio](https://user-images.githubusercontent.com/63621032/79176662-bdecde00-7df8-11ea-8ebe-b658d1ca767b.png)

Should I directly install from there?

Thank you.

**Comment by Genomics-CRT** [comment]
Clicking on that should in theory work yes but other people have been having issues so I'd suggest following this:

Create a new environment using Anaconda-navigator and select python and in the drop down box select python 3.7, also select R.

Once the environment is made go to home and click on install R-studio.

**Comment by JackSparrow1109** [comment]
Created environment and installed RStudio carefully, and they went well. Then I turned to the step of installing linux under Windows, finished step 3 ("This then starts loading (it might take a few minutes). When prompted to restart your computer, type in Y"). When I opened Anaconda-navigator again after laptop re-starting,  I find Jupyter notebook is installed as well under my environment as its button showing  "lauch":
![jupyter notebook](https://user-images.githubusercontent.com/63621032/79258824-a3137b80-7e83-11ea-8d1f-e0ca142301c2.png)
I'm sure didn't install Jupyter myself. What's up?
